certs/certs.go

```diff
@@ -10,7 +11,7 @@ import (
 	"github.com/deis/controller-sdk-go/api"
 )
 
-// List certs registered with the controller.
+// List lists certificates added to deis.
```

Comment:
SGTM, but @arschles was trying to have all our function names be nouns, not verbs, in the godoc hence the stutter.

WDYT in this circumstance @arschles?